apps/blazor-test-app/Properties/launchSettings.json

```diff
@@ -0,0 +1,25 @@
+{
+  "profiles": {
+    "Microsoft Teams (browser)": {
+      "commandName": "Project",
+      "launchBrowser": true,
+      "launchUrl": "https://teams.microsoft.com/l/app/0794bd56-95ae-4835-9588-d81b26ec3d8d?installAppPackage=true&webjoin=true&appTenantId=72f988bf-86f1-41af-91ab-2d7cd011db47",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      },
+      "dotnetRunMessages": "true",
+      "applicationUrl": "https://localhost:44302;http://localhost:2544",
+      "hotReloadProfile": "aspnetcore"
+    },
+    "WSL": {
+      "commandName": "WSL2",
+      "launchBrowser": true,
+      "launchUrl": "https://teams.microsoft.com/l/app/0794bd56-95ae-4835-9588-d81b26ec3d8d?installAppPackage=true&webjoin=true&appTenantId=72f988bf-86f1-41af-91ab-2d7cd011db47",
```

Comment:
Why do we need static `launchURL`?

Reply:
The static launchURL is used when debugging the Blazor app in Visual Studio. It allows for debugging the Blazor app inside of an instance of Teams web, rather than just the app on its own. So, it is not entirely necessary, as we are using the app on its own inside Orange for testing, but I feel it is nice to have the option to debug inside of Teams web, should the need arise.

Reply:
What if the same issue arises in Outlook or office? Also, I'm little bit worried about public display of `appTenantID` in the URL.

Reply:
Very well, I will remove it.